lib/libmpa/mpa_io.c

```diff
@@ -100,7 +101,8 @@ static int __mpa_digit_value(int c)
  *  Returns the maximum number of words needed to binary represent a number
  *  consisting of "digits" digits and each digits is in base "base".
  */
-static mpa_word_t __mpa_digitstr_to_binary_wsize_base_16(int digits)
+static
```

Comment:
The line is >80 characters, where would you split? After `__maybe_unused`? I could also shorten the function name.